sdk/storage_blobs/examples/list_containers_and_blobs.rs

```diff
@@ -0,0 +1,43 @@
+use azure_storage::core::prelude::*;
+use azure_storage_blobs::prelude::*;
+use futures::StreamExt;
+
+#[tokio::main]
+async fn main() -> azure_core::Result<()> {
+    env_logger::init();
+    // First we retrieve the account name, container and blob name from command line args
+
+    let account = std::env::args()
+        .nth(1)
+        .expect("please specify the account name as first command line parameter");
+
+    let account_key =
+        std::env::var("STORAGE_ACCOUNT_KEY").expect("Set env variable STORAGE_ACCOUNT_KEY first!");
+
+    let http_client = azure_core::new_http_client();
+    let storage_account_client =
+        StorageAccountClient::new_access_key(http_client, &account, &account_key);
+
+    let blob_service_client = storage_account_client.as_blob_service_client();
```

Comment:
In Cosmos, we would use the naming convention  `storage_account_client.blob_client()`. This was discussed in [a previous PR](https://github.com/Azure/azure-sdk-for-rust/pull/676#issuecomment-1062310005)

Reply:
I'll do this change as a follow-up PR, as I'd like feedback on it uniquely.